app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :author
```

Comment:
Kindly consider adding the class_name: 'User' to the author on this line to match the given ERD diagram. 

Eg: 
      `belongs_to :author, class_name: 'User'`

Reply:
done!